content/posts/newsletter-018/index.md

```diff
@@ -149,6 +149,26 @@ bringing more interesting crafting scenarios into the game.
 [Antorum]: https://ratwizard.dev/dev-log/antorum
 [@dooskington]: https://twitter.com/dooskington
 
+# [A/B Street]
```

Comment:
```suggestion
## [A/B Street]
```

Reply:
Whoops, thanks! I need to install mdlint or remember to check the CI